Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -287,6 +291,11 @@ static dispatch_queue_t globaVideoCaptureSerialQueue()
     if ([videoDevice position] == AVCaptureDevicePositionBack)
         capabilities.addFacingMode(VideoFacingMode::Environment);
 
+#if HAVE(AVCAPTUREDEVICE_MINFOCUSLENGTH)
+    double minimumFocusDistance = [videoDevice minimumFocusDistance] / 1000.0;
+    capabilities.setFocusDistance({ std::max<double>(minimumFocusDistance, 0), std::numeric_limits<double>::max() });
```

Comment:
Is 0 the maximum value, or the minimum?

Reply:
That is to prevent the -1 case (which means that min focus distance is unknown).
But we should probably do `setSupportsFocusDistance(true)` if we get -1, will change this.